src/gui/properties/peerlistwidget.cpp

```diff
@@ -382,47 +381,64 @@ void PeerListWidget::updatePeer(const BitTorrent::TorrentHandle *torrent, const
 {
     const PeerEndpoint peerEndpoint {peer.address(), peer.connectionType()};
     const QString peerIp = peerEndpoint.address.ip.toString();
+    const Qt::Alignment intDataTextAlignment = Qt::AlignRight | Qt::AlignVCenter;
+
+    const auto setModelData =
+        [this] (const int row, const int column, const QString &displayData
```

Comment:
Isn't possible to capture row like you do in another PR?

Reply:
no value of row will change depending on whether the peer is new or old, I can capture it by reference, but I don't think we want that

Reply:
You could have said that it is called with different arguments to clarify what is not clear from the diff, and I didn't have to go through all the method code after all.